definitions/features/redis.rb

```diff
@@ -0,0 +1,22 @@
+class Features::Redis < ForemanMaintain::Feature
+  SCL_PREFIX = 'rh-redis5'.freeze
+  SERVICE_NAME = "#{SCL_PREFIX}-redis".freeze
+
+  metadata do
+    label :redis
+
+    confine do
+      # Luckily, the service name is the same as the package providing it
+      server? && find_package(SERVICE_NAME)
+    end
+
+    def services
+      [system_service(SERVICE_NAME, 10)]
+    end
+
+    def config_files
+      ["/etc/opt/rh/#{SCL_PREFIX}/redis",
```

Comment:
I think this is a directory puppet-redis adds to be ready for multiple instances but we don't intend to use those. AFAIK the RPM doesn't create this directory. Debian(-based) does have such a directory.

Reply:
We should be safe, if a file does not exist, f-m will just skip it. I'd say it is better to try to collect everything there is